README.md

```diff
@@ -296,8 +296,9 @@ same major line. Should you need to upgrade to a new major, use an explicit
 - `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` are supported through
   [`node-proxy-agent`](https://github.com/TooTallNate/node-proxy-agent).
 
-- `COREPACK_INTEGRITY_KEYS` can be set to an empty string to instruct Corepack
-  to skip integrity checks, or a JSON string containing custom keys.
+- `COREPACK_INTEGRITY_KEYS` can be set to an empty string, `0`, or `false` to
```

Comment:
nit: we don't support using `'false'` for any other env variable, I think that would be confusing
```suggestion
- `COREPACK_INTEGRITY_KEYS` can be set to an empty string or `0` to
```
Or we could have a specific magic string to avoid any ambiguity such as:
```suggestion
- `COREPACK_INTEGRITY_KEYS` can be set to `'skip_check'` to
```

Reply:
In this case, having the possibility of setting `0` is more aligned with the other configurations.